TESTS/network/wifi/wifi_connect_secure.cpp

```diff
@@ -34,6 +34,15 @@ void wifi_connect_secure(void)
     TEST_ASSERT_EQUAL_INT(NSAPI_ERROR_OK, wifi->connect());
 
     TEST_ASSERT_EQUAL_INT(NSAPI_ERROR_OK, wifi->disconnect());
+
+    // Driver shall cache the credentials
```

Comment:
How about we replace the old code with the new code, instead of connecting twice?

Reply:
That makes sense, fixed.